src/pages/Introduction/Introduction.scss

```diff
@@ -1,5 +1,6 @@
 .introduction {
     height: 100vh;
```

Comment:
Consider using descriptive class names for better maintainability and understanding.

Suggested change:
```scss
.introduction {
    height: 100vh;
    background-color: #1c1031;
}
``` 

For reference, see [CSS Naming Conventions](https://cssguidelin.es/#css-naming-conventions).